### DISCUSSION TITLE
/cmi sudo allplayer c:

### DISCUSSION THREAD

**Discussion by Arkhe002** [Zrips/CMI#6816] [Feature Requests]
Rather than entering individual player names, such a command would be very helpful.

**Comment by ParrotRacer** [comment]
allPlayers! cmi sudo [playerName] -s

**Reply by CloudeLecaw** [reply]
> allPlayers! cmi sudo [allPlayers] c:Hello All -s

You can use asConsole! before if it comes from a custom alias so exectuee something they normaly cant or so.. well mostly ist s just to sudo all players to do something like /spawn (what also could be done by /tpa.. what ever,,
Also here instead of [playerName] use [allPlayers]

Note Sudo might has the little probblem it will not effect all staff members. When they have the sudo bypass permisison then sudo has no effect on them.

**Comment by CloudeLecaw** [comment]
`allPlayers! cmi sudo [allPlayers] say hello`

`allPlayers! cmi sudo [allPlayers] c:Greetings everyone`

**Comment by mrfloris** [comment]
Here's an example of using the regular features of CMI to make global commands;

https://github.com/mrfdev/CMI/tree/master/Resources/Add-ons/globalcommand

You can try to use this to achieve what you want; if you have any contributions feel free to clone and PR, and I will review/merge with my addon so others can also benefit from your contributions.
